articles/virtual-machines/workloads/sap/azure-monitor-sap-quickstart.md

```diff
@@ -80,11 +80,21 @@ Sign in to the Azure portal at https://portal.azure.com
 
 > [!IMPORTANT]
 > To configure OS (Linux) provider, ensure that Node_Exporter is installed in each BareMetal instance. For more information, see [Node_Exporter](https://github.com/prometheus/node_exporter)
+> ALso download the Node Exporter from [this](https://prometheus.io/download/#node_exporter) link only.
```

Comment:
A minor observation: L is mistakenly capitalised in ALso

Reply:
Fixed